tensorflow_addons/metrics/BUILD

```diff
@@ -26,3 +26,28 @@ py_test(
         ":metrics",
     ],
 )
+
+py_library(
```

Comment:
Hey, this is not right. You need to include `r_square.py` under the already defined build rule.